registry/storage/filewriter.go

```diff
@@ -72,7 +72,7 @@ func newFileWriter(ctx context.Context, driver storagedriver.StorageDriver, path
 	buffered := bufferedFileWriter{
```

Comment:
@stevvooe  I just tried that change, but it still fails the modified test case. Does it work for you? Perhaps, I am missing something.